README.md

```diff
@@ -86,301 +77,121 @@ database.
-of the Open vSwitch kernel module that is compiled from this repo.  If you use
-the Open vSwitch kernel module from upstream Linux, you will need a minumum
-kernel version of 3.18 for geneve.  There is no stt support in upstream Linux.
-You can verify whether you have the support in your kernel by doing a `lsmod |
-grep $ENCAP_TYPE`.)
-
-```
-ovs-vsctl set Open_vSwitch . external_ids:ovn-remote="tcp:$CENTRAL_IP:6642" \
-  external_ids:ovn-nb="tcp:$CENTRAL_IP:6641" \
-  external_ids:ovn-encap-ip=$LOCAL_IP \
-  external_ids:ovn-encap-type="$ENCAP_TYPE"
-```
-
-In addition, each Open vSwitch instance in an OVN deployment needs a unique,
-persistent identifier, called the "system-id".  If you install OVS from
-distribution packaging for Open vSwitch (e.g. .deb or .rpm packages), or if
-you use the ovs-ctl utility included with Open vSwitch or the startup
-scripts that come with Open vSwitch, it automatically configures a system-id.
-If you start Open vSwitch manually, you should set one up yourself.
-
-For example:
-
-```
-id_file=/etc/openvswitch/system-id.conf
-test -e $id_file || uuidgen > $id_file
-ovs-vsctl set Open_vSwitch . external_ids:system-id=$(cat $id_file)
-```
-
-And finally, start the ovn-controller.  (You need to run the below command on
-every boot)
+Also start ovn-controller on this node.
 
 ```
 /usr/share/openvswitch/scripts/ovn-ctl start_controller
 ```
 
-### k8s master node initialization.
+Now start the ovnkube utility on the master node too.
 
-Note: The init scripts used below use fair defaults.  If you are an advanced
-user, please read [config.md] for more fine grained control.
+The below command expects the user to provide
+* A cluster wide private address range of $CLUSTER_IP_SUBNET
+(e.g: 192.168.0.0/16).  The pods are provided IP address from this range.
 
-Set the k8s API server address in the Open vSwitch database for the
-initialization scripts (and later daemons) to pick from.
-
-```
-ovs-vsctl set Open_vSwitch . external_ids:k8s-api-server="127.0.0.1:8080"
-```
+* $NODE_NAME should be the same as the one used by kubelet.  kubelet by default
+uses the hostname.  kubelet allows this name to be overridden with
+--hostname-override.
 
-Clone this repository and install the executables.
+* $SERVICE_IP_SUBNET is the same as the one provided to k8s-apiserver via
+--service-cluster-ip-range option. An e.g is 172.16.1.0/24.
 
 ```
-git clone https://github.com/openvswitch/ovn-kubernetes
-cd ovn-kubernetes
-pip install .
-```
-
-On the master node, with a unique node name of $NODE_NAME, for a cluster wide
-private address range of $CLUSTER_IP_SUBNET and a smaller subnet for the
-logical switch created in the master node of $MASTER_SWITCH_SUBNET, run:
-
-```
-ovn-k8s-overlay master-init \
-  --cluster-ip-subnet=$CLUSTER_IP_SUBNET \
-  --master-switch-subnet="$MASTER_SWITCH_SUBNET" \
-  --node-name="$NODE_NAME"
+ sudo ovnkube -k8s-kubeconfig kubeconfig.yaml -net-controller \
```

Comment:
Is it worth providing a link to the sample kubeconfig.yaml here?

Reply:
It is probably a overkill as people that start a kubernetes cluster know this.